tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -1686,22 +1683,19 @@ static final boolean userCanDeleteUser(User managerUser, int targetId, int targe
     }
 
     static final boolean userCanDoList(User managerUser) {
-        return (userIsVetter(managerUser));
+        return managerUser.getLevel().canDoList();
     }
 
     public static final boolean userCanCreateUsers(User u) {
-        return (userIsTC(u) || userIsExactlyManager(u));
+        return u.getLevel().canCreateUsers();
     }
 
     static final boolean userCanSubmit(User u) {
```

Comment:
This method -- with only one arg -- appears to be unused, in which case it should be removed

Reply:
Could be. For near future ?